en/config/airspeed.md

```diff
@@ -17,9 +15,6 @@ Before calibration they must be [enabled via the corresponding parameter](../adv
 
 ## Performing the Calibration
 
-The airspeed calibration needs to read a stable baseline with 0 airspeed in order to determine an offset.
```

Comment:
Just not useful?

Reply:
That and that I think it can be miss-leading - it doesn't have to be exact 0, just to lower the better. 0 is quite impossible to achieve, there's always some noise on it.